.env.example

```
@@ -55,6 +55,8 @@ BETTER_AUTH_SECRET=replace-with-a-different-32-character-random-secret
 # SUPERADMIN in PostgreSQL; either gate alone is insufficient.
 SUPERADMIN_EMAILS=
 RESEND_API_KEY=
+# Svix-compatible signing secret from the enabled Resend delivery webhook.
+RESEND_WEBHOOK_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RESEND_WEBHOOK_SECRET key should go before the SUPERADMIN_EMAILS key

(UnorderedKey)